honeybee/radiance/datatype.py

```diff
@@ -51,36 +98,50 @@ def __set__(self, instance, value):
                                     ",".join(map(str, inputs)), value))
             setattr(instance, self._name, value)
 
-class RadDescrBoolFlags(RadDescrDefault):
+
+class RadianceBoolFlag(RadianceDefault):
     """Descriptor for all the inputs that are boolean flags or single
     alphabets
     """
     pass
 
 
-class RadDescrNum(RadDescrDefault):
+class RadianceNumber(RadianceDefault):
     """
     Descriptor to set numbers.
+    (Attributes inherited from base-class are explained there.)
```

Comment:
The docstring will show up for developers and in documentation so we need to have the arguments in both places. It's a little bit of duplication but it does worth it if we consider the fact that we will be generating all the documentation from the source code.